### PR TITLE
Update the browse and welcome page functionality.

### PR DESCRIPTION
**Pull request by jKwibe**

### CODE REVIEW

File: app/views/partials/_navbar.html.erb [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <% if current_user %>
       <ul class="navbar-nav current-user-nav">
         <li class="nav-item">
-          <%= link_to 'Browse Music','#', class: "nav-link" %>
+          <%= link_to 'Browse Music',browse_index_path, class: "nav-link" %>
```
Comment (twarbelow) [inline app/views/partials/_navbar.html.erb]: please add a space after the comma before browse index path

File: spec/features/welcome/index_spec.rb [hunk 2]
```diff
@@ -1,15 +0,0 @@
-require 'rails_helper'
```
Comment (twarbelow) [inline spec/features/welcome/index_spec.rb:1]: why is this deleted?

Comment (jKwibe) [reply spec/features/welcome/index_spec.rb:1]: we didn't need it because the only thing that the test was checking was the login by google button which we removed.

Comment (twarbelow) [reply spec/features/welcome/index_spec.rb:1]: ok
